applications/CSharpWrapperApplication/tests/cpp_tests/test_calculate.cpp

```diff
@@ -102,9 +103,9 @@ namespace Kratos {
 
             // Import mdpa
             CreateMDPAFile();
-            const std::string file_name = FilesystemExtensions::JoinPaths({FilesystemExtensions::CurrentWorkingDirectory(), "file.mdpa"});
+            std::filesystem::path file_path = std::filesystem::current_path() / "file.mdpa";
             CSharpKratosWrapper::KratosWrapper *wrapperInstance = new CSharpKratosWrapper::KratosWrapper();
-            wrapperInstance->init(file_name.c_str());
+            wrapperInstance->init(file_path.string().c_str());
```

Comment:
will this work? Remember that `std::filesystem::path::string()` returns an `std::wstring` instead of a normal `std::string` on Windows.